### PR TITLE
Add BVA Intake Queue

### PR DESCRIPTION
**Pull request by Jeremy1026**
Resolves CASEFLOW-2276

### Description
Adds new custom BVA Intake Queue to Caseflow.

### Acceptance Criteria
- [ ] BVA Intake org has a custom queue which includes a tab to show all cases in the pre-docket workflow
- [ ] The new tab is is given the following columns:
         Case Details
          Task owner (e.g. which organization currently has the open tasks)
          Tasks
          Days since last status change
          Total Days

### CODE REVIEW

File: app/models/queue_tabs/bva_intake_completed_tab.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    COPY::ORGANIZATIONAL_QUEUE_COMPLETED_TAB_TITLE
+  end
+
+  def self.tab_name
+    Constants.QUEUE_CONFIG.COMPLETED_TASKS_TAB_NAME
+  end
+
+  def description
+    format(COPY::USER_QUEUE_PAGE_ASSIGNED_TASKS_DESCRIPTION, assignee.name)
+  end
```
Comment (ajspotts) [inline app/models/queue_tabs/bva_intake_completed_tab.rb:18]: Need to update the COPY to reflect designs "Cases that are complete:"

File: app/models/queue_tabs/bva_intake_pending_tab.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    COPY::ORGANIZATIONAL_QUEUE_PAGE_PENDING_TAB_TITLE
+  end
+
+  def self.tab_name
+    Constants.QUEUE_CONFIG.PENDING_TASKS_TAB_NAME
+  end
+
+  def description
+    format(COPY::USER_QUEUE_PAGE_ASSIGNED_TASKS_DESCRIPTION, assignee.name)
+  end
```
Comment (ajspotts) [inline app/models/queue_tabs/bva_intake_pending_tab.rb:18]: Same thing here: "Cases that are pending VHA action:"

File: app/models/queue_tabs/bva_intake_ready_for_review_tab.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    COPY::ORGANIZATIONAL_QUEUE_PAGE_READY_FOR_REVIEW_TAB_TITLE
+  end
+
+  def self.tab_name
+    Constants.QUEUE_CONFIG.READY_FOR_REVIEW_TASKS_TAB_NAME
+  end
+
+  def description
+    format(COPY::USER_QUEUE_PAGE_ASSIGNED_TASKS_DESCRIPTION, assignee.name)
+  end
```
Comment (ajspotts) [inline app/models/queue_tabs/bva_intake_ready_for_review_tab.rb:18]: And here: "Cases assigned to Board Intake:"

File: client/app/queue/utils.js [hunk 4]
```diff
@@ -72,6 +72,7 @@ export const prepareMostRecentlyHeldHearingForStore = (appealId, hearing) => {
 };
 
 const taskAttributesFromRawTask = (task) => {
+  console.log("LOGGING 2: ",task)
```
Comment (ajspotts) [inline client/app/queue/utils.js:75]: probably want to remove this

File: app/models/queue_tabs/bva_intake_ready_for_review_tab.rb (hunk 3 above)
Comment (Jeremy1026) [reply app/models/queue_tabs/bva_intake_ready_for_review_tab.rb:18]: Updated copy

File: app/models/queue_tabs/bva_intake_pending_tab.rb (hunk 2 above)
Comment (Jeremy1026) [reply app/models/queue_tabs/bva_intake_pending_tab.rb:18]: Updated copy

File: app/models/queue_tabs/bva_intake_completed_tab.rb (hunk 1 above)
Comment (Jeremy1026) [reply app/models/queue_tabs/bva_intake_completed_tab.rb:18]: Updated copy

File: client/app/queue/utils.js (hunk 4 above)
Comment (Jeremy1026) [reply client/app/queue/utils.js]: Removed

**Review by ajspotts — APPROVED** [review]

**Pull request merged**